src/stems/stem-tracker.js

```diff
@@ -21,49 +22,48 @@ function StemTracker(editor, onClick) {
 
   /** Listen for changes in content and update. */
   editor.addListener({
-    onContent: function () {
-      updateStems( this.editorElem );
+    onKeyup: function (e) {
```

Comment:
did onSelection not work well enough, need onKeyup + onMouseup?

Did you want to implement the hover behaviour, or just have the stem only follow the actual cursor.